web/src/components/ArrowTooltip.test.tsx

```diff
@@ -25,7 +26,7 @@ const renderToolTip = (): void => {
     <ThemeProvider theme={createTheme()}>
       <ArrowTooltip title={testString}>
         <div className="fdr fac font-body-lg text-green" data-testid="tooltip">
-          <Icon>help_outline</Icon>
+          <HelpOutline />
```

Comment:
[pebble] @Sun-Mountain Is there a reason we opted for this over using the SVG sheet the way the other icons are being set?

I honestly don't know off the top of my head if the SVG sheet is from MUI or NJWDS.

In either case I'd opt for keeping this consistent and changing this line to
```js
<Icon iconName="help_outline"/>
```
for consistency sake if nothing else.

Reply:
This one was using the MUI Icon and not the one from the NJ library. It was the first one I found so that's probably why it got lost in the sauce. Will fix!